src/winter_onboarding_2021/fleet_management_service/handlers/cab.clj

```diff
@@ -32,3 +35,11 @@
     "Add a cab"
     (cab-views/cab-form
      (get-in request [:flash :data])))))
+
+(defn view-cab [request]
+  (let [cab (cab-model/get-by-id (get-in request [:params :slug]))]
```

Comment:
What's the `slug`?

We should also support get cab by id.

Reply:
`slug` is just `id`. I should name it better to `id`.